appendices/appendix05-fmd.tex

```diff
@@ -0,0 +1,14 @@
+% !TEX root = ../zeth-protocol-specification.tex
+
+\chapter{Fuzzy message detection}\label{appendix:fmd}
+
+As explained in~\ref{zeth-protocol:zeth-receive} and~\ref{client-security:syncing}, in order to receive $\zethnotes$, a $\zeth$ user must listen on an broadcast channel, and try to decrypt all encrypted events emitted by the $\mixer$ contract. While providing the best potential for indistinguishability (all users scan the chain data and expose the same behavior), such routine is particularly expensive to carry out, especially for computationally restricted users (i.e.~users with computationally limited devices).
```

Comment:
> in order to receive $\zethnotes$, a $\zeth$ user must listen on an broadcast channel
->
in order to receive $\zethnotes$, a $\zeth$ user must listen on a broadcast channel